client_end_user/shared/src/commonMain/kotlin/resources/BpResources.kt

```diff
@@ -0,0 +1,46 @@
+package resources
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.ReadOnlyComposable
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.staticCompositionLocalOf
+import com.beepbeep.designSystem.ui.theme.BpTheme
+import com.seiko.imageloader.LocalImageLoader
+import util.generateImageLoader
+import util.getPlatformContext
+
+private val localDrawableResources = staticCompositionLocalOf { DrawableResources() }
+private val localStringResources = staticCompositionLocalOf { StringResources() }
+
+@Composable
+fun BeepBeepTheme(
+    useDarkTheme: Boolean = isSystemInDarkTheme(),
+    content: @Composable () -> Unit,
+) {
+    val drawableResources = if (useDarkTheme) BpDrawableDarkResources else DrawableResources()
+    val context = getPlatformContext()
+
+    CompositionLocalProvider(
+        localDrawableResources provides drawableResources,
+        localStringResources provides StringResources(),
+        LocalImageLoader provides remember { generateImageLoader(context) }
```

Comment:
as Barq said we need to delete LocalImageLoader from here

Reply:
Done